### PR TITLE
miller: add caveat message

### LINKED ISSUE #343: brew installation message
I installed miller using Homebrew for macOS.  The output was:

```
$ brew install miller
==> Downloading https://homebrew.bintray.com/bottles/miller-5.7.0.mojave.bottle.tar.gz
==> Downloading from https://akamai.bintray.com/…
==> Pouring miller-5.7.0.mojave.bottle.tar.gz
🍺  /Local/Cellar/miller/5.7.0: 6 files, 887.7KB
```

I was disappointed that the installation didn't finish with a message like:

```
🍻 It's Miller Time! 🍻 
```

I don't know whether that would be possible, but it'd be very cool.  I know, it would probably involve a change to the Homebrew formula, but I figure the maintainers of this repo are probably the best ones to lobby for this change.

Good luck!

### PR DESCRIPTION
**Pull request by sloanlance**
In response to johnkerl/miller#343 (approved by @johnkerl), add a humorous message that's shown to the user when they install Miller.  (Or when they request info about the formula.)

Resolves johnkerl/miller#343.

- [x] Have you followed the [guidelines for contributing](https://github.com/Homebrew/homebrew-core/blob/HEAD/CONTRIBUTING.md)?
- [X] Have you checked that there aren't other open [pull requests](https://github.com/Homebrew/homebrew-core/pulls) for the same formula update/change?
- [X] Have you built your formula locally with `brew install --build-from-source <formula>`, where `<formula>` is the name of the formula you're submitting?
- [X] Is your test running fine `brew test <formula>`, where `<formula>` is the name of the formula you're submitting?
- [x] Does your build pass `brew audit --strict <formula>` (after doing `brew install <formula>`)?

-----

### CODE REVIEW

File: Formula/miller.rb
Comment (chenrui333) [inline Formula/miller.rb:34]: Nice, ❤️

Comment (sloanlance) [reply Formula/miller.rb:34]: Thanks!

Are you able to approve this PR, or do I need to do something more first?

Comment (chenrui333) [reply Formula/miller.rb:34]: Yeah, sure, can you squash the commits into one with message as `miller: add caveat message`, I think it is good to 🚀 ! Thanks!

Comment (sloanlance) [reply Formula/miller.rb:34]: I'm used to squashing commits during the PR merge step, but it seems like you're asking me to do it before then.  What's a good way to do that?  Would it require opening a new PR?

Comment (gromgit) [reply Formula/miller.rb:34]: See https://docs.brew.sh/How-To-Open-a-Homebrew-Pull-Request#following-up, specifically the section that begins "To make changes based on feedback".

**Review by SMillerDev — CHANGES_REQUESTED** [review]
This is not what caveats are for.

**Comment by sloanlance** [comment]
> This is not what caveats are for.

OK.  So, where can we see the specifications for caveat contents?

**Comment by miccal** [comment]
> OK. So, where can we see the specifications for caveat contents?

[See here](https://docs.brew.sh/Formula-Cookbook#caveats).

**Comment by sloanlance** [comment]
> [See here](https://docs.brew.sh/Formula-Cookbook#caveats).

Thanks!  I'll check with @johnkerl, the Miller author/maintainer, and see whether there are any specific messages that would apply that could appear in the caveat of its formula.  In the meantime, can this PR  remain in an "on hold" state?

**Pull request closed without merging**
